### PR TITLE
2 fixes: member json-file typo. Can now render PR to non-guild member…

### PR DESCRIPTION
**Pull request by afrxx09**
… repo

### CODE REVIEW

File: src/data/members.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ Check users/krawaller.json for an example on what your file should look like!
 
 var members = {
 	krawaller: require("./users/krawaller.json"),
-	Dagashi: require("./users/Dagashi.json"),
+	Dagashi: require("./users/dagashi.json"),
```
Comment (krawaller) [inline src/data/members.js:8]: Is this change intentional? Pretty sure it should be with capital D.

**Comment by Pajn** [comment]
The file is with lowercase, as Andreas runs Linux it broke. Doesn't matter on other platforms

**Comment by krawaller** [comment]
Huh, you're right. It's uppercase in my local repo, but that's git not telling the difference.
